pkg/virtctl/create/vm/vm_test.go

```diff
@@ -616,6 +670,21 @@ var _ = Describe("create vm", func() {
 			Entry("Bootorder set to 0", "src:my-ds,bootorder:0", "failed to parse \"--volume-datasource\" flag: bootorder must be greater than 0"),
 		)
 
+		DescribeTable("Invalid arguments to VolumeImportFlag", func(params, errMsg string) {
+			out, err := runCmd(setFlag(VolumeImportFlag, params))
+
+			Expect(err).To(MatchError(errMsg))
+			Expect(out).To(BeEmpty())
+		},
+			Entry("Missing size with blank source", "type:blank", "failed to parse \"--volume-import\" flag: size must be specified"),
```

Comment:
Can you add a negative test case for an invalid `PullMethod` when using the registry import?